### DISCUSSION TITLE
Pose holder for rigs

### DISCUSSION THREAD

**Discussion by benngarnish** [jjburton/cgmTools#205] [Ideas]
Adding a pose switcher at the rig level. 
- T Pose
- A pose 
- Sim pose

This would be a switch that reposes your character but without adding values to the rig controllers. 

Any suggestions here on how to handle this?

**Comment by jjburton** [comment]
This is doable  but would take some thinking through. You'd want to have your target values as world vectors to align to more than likely. With shoulders you'd want the clav rotated more than likely for a better t it was modeled in A-pose.  Maybe take this to a discussion to talk through?
